src/main/kotlin/com/goodchoice/domain/review/model/ReviewAuthor.kt

```diff
@@ -0,0 +1,5 @@
+package com.goodchoice.domain.review.model
+
+import java.util.*
+
+data class ReviewAuthor(val id: UUID, val firstName: String, val lastName: String)
```

Comment:
Maybe rename to `UserPreview` and move to `user` package? This model can be used for any user role in different scenarios

Reply:
done